src/mscorlib/corefx/System/Globalization/Calendar.cs

```diff
@@ -840,7 +840,6 @@ internal static long TimeToTicks(int hour, int minute, int second, int milliseco
             throw new ArgumentOutOfRangeException(null, SR.ArgumentOutOfRange_BadHourMinuteSecond);
         }
 
-        [System.Security.SecuritySafeCritical]  // auto-generated
```

Comment:
These attributes are ignored in both CoreCLR and CoreRT - they were physically removed in CoreRT.